htcanalyze/main.py

```diff
@@ -646,21 +657,26 @@ def print_results(htcanalyze: HTCAnalyze,
 
 def run(commandline_args):
     """
-    Run this script.
+    Run HTCAnalyze
+
+    read command line arguments to set the analyzing-mode
+    and set parameters for finding and reading the input files,
+    set the output streams and manage the format of the output.
 
     :param commandline_args: list of args
     :return:
     """
     # before running make sure Global Parameters are set to default
-    GlobalServant.reset()
+    GlobalStream.reset()
 
     if not isinstance(commandline_args, list):
         commandline_args = commandline_args.split()
 
     try:
-        start = date_time.now()  # start date for runtime
+        # save start time of script
```

Comment:
I thought this was already done at the top of the script. Is this one different?

Reply:
This one actually saves the start date for runtime ;) But any comments here are superfluous. How is `start = datetime.datetime.now()` now clearly indicating a start time?

Also, don't put comments _below_ the line they are relevant for, put them _above_. Furthermore, a comment on the same line as the code is fine, especially if the comment is brief. It just needs to have two spaces between the end of the code and the start of the comment.